### ISSUE TITLE
KDE seems to be deprecating kdesu, replaced with kdesudo

### ISSUE THREAD

**Issue by rubin110** [sinkuu/atom-susave#5]
https://blog.martin-graesslin.com/blog/2017/02/editing-files-as-root/

The killing a fly with a hammer post doesn't directly relate to kdesu, however the comments lead to me believe that kdesu has also been deprecated and replaced with kdesudo.

Any chance kdesudo could be added to the list of available sudo guis?

Thanks!

**Comment by sinkuu** [comment]
Hmm, is `kdesudo` actively maintained somewhere? Also recent KDE distribution contains `kdesu` but not `kdesudo`.

https://cgit.kde.org/kdesu.git/
https://cgit.kde.org/kdesudo.git/

**Comment by stikonas** [comment]
No, kdesu is not being replaced with kdesudo. In fact functionality of kdesudo was merged into kdesu long time ago.  Running GUI as root is being deprecated.
